hdp/src/primitives/chain_id.rs

```diff
@@ -11,6 +11,7 @@ pub enum ChainId {
     EthereumSepolia,
     StarknetMainnet,
     StarknetSepolia,
+    Unknown,
```

Comment:
do we need to have `Unknown` value? isn't this will handled while parsing as it's enumerated ?

Reply:
It was to have a default destination chain id that is not opinionated (e.g., to default to ETHEREUM_SEPOLIA) in the default implementation of the CLI config.
If that's acceptable to have the default chain id being Ethereum Sepolia then I can safely remove, wdyt?

Reply:
i guess we should make it fail if user passed not supported chain id as cli config, i agree with point of having specific chain as default is bad, but we also dont wanna have additional branch of chain id enum

Reply:
*Removed*